pkg/services/sqlstore/migrations/user_auth_token_mig.go

```diff
@@ -32,4 +32,16 @@ func addUserAuthTokenMigrations(mg *Migrator) {
 	mg.AddMigration("add unique index user_auth_token.prev_auth_token", NewAddIndexMigration(userAuthTokenV1, userAuthTokenV1.Indices[1]))
 
 	mg.AddMigration("add index user_auth_token.user_id", NewAddIndexMigration(userAuthTokenV1, userAuthTokenV1.Indices[2]))
+
+	mg.AddMigration(
+		"Add revoked_at to the user auth token",
+		NewAddColumnMigration(
+			userAuthTokenV1,
+			&Column{
+				Name:     "revoked_at",
+				Type:     DB_Int,
+				Nullable: true,
+			},
+		),
+	)
```

Comment:
@bergquist new db migration coming!

Reply:
Is this okay? 🤔  Something should I consider before merging? Thanks!